Website/admin/ControlPanel/AddModule.ascx.cs

```diff
@@ -495,6 +509,13 @@ private void DoAddExistingModule(int moduleId, int tabId, string paneName, int p
                 }
                 else
                 {
+                    //copy tab module settings
+                    newModule.TabModuleSettings.Clear();
```

Comment:
I believe the settings should be copied only when copy checkbox is checked. So there is not need to copy settings in else part.